common/core/github.com/juju/juju/cmd/juju/commands/bootstrap-fakejuju.go

```diff
@@ -0,0 +1,91 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/juju/errors"
+
+	"github.com/juju/juju/jujuclient"
+	"github.com/juju/juju/testing"
+	"github.com/juju/juju/version"
+)
+
+// Custom bootstrap logic for fake juju, which essentially:
+//
+// - populates the JUJU_DATA directory with files pointing to the
+//   fake-jujud process, as identified by the port number in the
+//   FAKE_JUJUD_PORT environment variable.
+//
+// - connects to fake-jujud an get the agent version to ensure that
+//   all parameters are correct.
+//
+// Once the bootstrap is complete, the CLI can be used like a regular
+// juju one.
+func (c *bootstrapCommand) fakeJujuBootstrap() error {
+	store := c.ClientStore()
+	controller := c.controllerName
+	model := "controller"
+
+	if err := writeControllersFile(store, controller); err != nil {
+		return err
+	}
+
+	if err := writeAccountsFile(store, controller); err != nil {
+		return err
+	}
+
+	if err := writeModelsFile(store, controller, model); err != nil {
+		return err
+	}
+
+	if err := store.SetCurrentModel(controller, model); err != nil {
+	}
```

Comment:
Did you want to do the same as all the above if blocks? If not, perhaps not even check for err and just call store.SetCurrentModel() directly.